src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -267,8 +286,10 @@ internal unsafe int IndexOfCore(string source, string target, int startIndex, in
         }
 
         // For now, this method is only called from Span APIs with either options == CompareOptions.None or CompareOptions.IgnoreCase
-        internal unsafe int IndexOfCore(ReadOnlySpan<char> source, ReadOnlySpan<char> target, CompareOptions options, int* matchLengthPtr)
+        internal unsafe int IndexOfCore(ReadOnlySpan<char> source, ReadOnlySpan<char> target, CompareOptions options, int* matchLengthPtr, bool start)
         {
+            // TODO: Add LastIndexOf --> start = false impl
```

Comment:
that's the work in progress.

Reply:
I don't understand.  This PR isn't ready for review yet?

Reply:
right, when I pushed the changes I discovered I forgot a branch on Unix which then failed in CI. I now flagged the PR as work in progress.